lib/alephant/logger/cloudwatch.rb

```diff
@@ -3,8 +3,11 @@
 module Alephant
   module Logger
     class CloudWatch
-      def initialize(namespace)
-        @namespace = namespace
+      def initialize(namespace, unit = "Count", value = 1, dimensions = {})
+        @namespace  = namespace
+        @unit       = unit
+        @value      = value
```

Comment:
Would suggest these go into a `default` instance variable hash to clarify intent